nomad-gcp/README.md

```diff
@@ -29,7 +29,7 @@ output "module" {
 }
 ```
 
-Use latest codebase:
+Using main branch codebase:
```

Comment:
I am confused by this line

This is a nit and scope creep so don't feel the need to act as part of this PR, but I am unclear why there are 2 examples in this readme.  If we need another example of autoscaling or RBAC, it should be in the examples directory.